src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseConcreteTypeAnalyzer.cs

```diff
@@ -116,7 +116,14 @@ public override void Initialize(AnalysisContext context)
                 var voidType = context.Compilation.GetSpecialType(SpecialType.System_Void);
                 context.RegisterSymbolStartAction(context =>
                 {
-                    var coll = Collector.GetInstance(voidType);
+                    var namedType = (INamedTypeSymbol)context.Symbol;
+                    if (namedType.TypeKind == TypeKind.Interface)
+                    {
+                        // nothing to do here
+                        return;
+                    }
+
+                    var coll = Collector.GetInstance(voidType, symbol => context.Options.MatchesConfiguredVisibility(UseConcreteTypeForMethodReturn, symbol, context.Compilation, SymbolVisibilityGroup.Private));
```

Comment:
This should be documented in https://github.com/dotnet/roslyn-analyzers/blob/main/docs/Analyzer%20Configuration.md#supported-editorconfig-options

Reply:
Done.